deploy/dracon/templates/job.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: batch/v1
```

Comment:
i think we should add a boolean to enable migrations and have the entire job in an if so that if for whatever reason the migrations are not supposed to be deployed, they can be skipped